sources/ConflictWindow.cpp

```diff
@@ -19,7 +19,7 @@ static const unsigned kReplace = 'RPLC';
 static const unsigned kSkip = 'SKIP';
 
 ConflictWindow::ConflictWindow(const char* file)
-	: BWindow(BRect(0, 0, 0, 0), "", B_BORDERED_WINDOW, B_WILL_ACCEPT_FIRST_CLICK | B_AUTO_UPDATE_SIZE_LIMITS)
+	: BWindow(BRect(0, 0, 0, 0), "", B_MODAL_WINDOW, B_WILL_ACCEPT_FIRST_CLICK | B_AUTO_UPDATE_SIZE_LIMITS)
```

Comment:
I found it's too easy to "lose" the window under others. What do you think about these window flags:

```
ConflictWindow::ConflictWindow(const char* file)
	: BWindow(BRect(0, 0, 0, 0), "AutoFiler: Conflict", B_FLOATING_WINDOW_LOOK,
		B_FLOATING_ALL_WINDOW_FEEL,
		B_NOT_CLOSABLE | B_NOT_ZOOMABLE | B_NOT_RESIZABLE | B_AVOID_FOCUS
			| B_AUTO_UPDATE_SIZE_LIMITS)

```
It has a small window tab for easy moving, floats over all windows but doesn't steal focus in case you're working in some window but didn't notice the Conflict popping up.